genie-web/src/main/java/com/netflix/genie/web/controllers/UIController.java

```diff
@@ -69,9 +71,19 @@ public String getIndex(@NotNull final HttpServletResponse response, @Nullable fi
      * @param id The id of the job
      * @param request the servlet request to get path information from
      * @return The forward address to go to at the API endpoint
+     * @throws UnsupportedEncodingException if URL-encoding of the job id fails
      */
     @GetMapping(value = "/file/{id}/**")
-    public String getFile(@PathVariable("id") final String id, final HttpServletRequest request) {
-        return "forward:/api/v3/jobs/" + id + "/" + ControllerUtils.getRemainingPath(request);
+    public String getFile(
+        @PathVariable("id") final String id,
+        final HttpServletRequest request
+    ) throws UnsupportedEncodingException {
```

Comment:
We don't have an exception mapper for this exception do you know what error code it maps to by default? 500 I assume?